pyreal/explainers/generic_explainer.py

```diff
@@ -0,0 +1,119 @@
+from pyreal.explainers import BaseExplainer, ShapFeatureImportance
+
+
+def explainer(return_explanation=True, return_explainer=True, explainer=None,
```

Comment:
Maybe rename to `explainer`, to use functional language

Reply:
Do you mean `generic_explainer`?

Reply:
whoops, I meant `explain`